### ISSUE TITLE
Migration problem

### ISSUE THREAD

**Issue by kultz** [laravelio/laravel.io#228]
After running migration this happened:

> Migration table created successfully.
>                                                                           
>   [Illuminate\Database\QueryException]                                                                                 
>   SQLSTATE[42000]: Syntax error or access violation: 1101 BLOB/TEXT column 'description' can't have a default value (  
>   SQL: alter table `tags` add `description` text not null default '')                                                  
>                                                                                                                        
> 
>                                                                                       
>   [PDOException]                                                                                                     
>   SQLSTATE[42000]: Syntax error or access violation: 1101 BLOB/TEXT column 'description' can't have a default value
> 

Please help me with this.

**Comment by kultz** [comment]
Fixed this by changing these in "2013_09_23_111349_add_description_field_to_tags.php" :

```
    public function up()
    {
        Schema::table('tags', function(Blueprint $table) {
            $table->text('description')->default('');
        });
    }
```

> $table->text('description')->default('') to $table->text('description')->nullable()

**Comment by driesvints** [comment]
Yep, I stumbled on the same problem a few months back. Seems like something changed in the default behaviour for this. Feel free to send in a PR :)

**Comment by driesvints** [comment]
Fixed in #229
